modules/mcagar/src/main/ts/ephox/mcagar/api/TinyHooks.ts

```diff
@@ -53,10 +53,14 @@ const setupHooks = <T extends EditorType = EditorType>(
       teardownEditor();
       teardown();
     }
+
+    // Cleanup references
+    editor = undefined;
+    teardownEditor = Fun.noop;
   });
 
   return {
-    editor: () => editor
+    editor: () => editor as T
```

Comment:
Do we want to assert that this is non-nullable before we return it, just so that we can get a hard error in case someone somehow manages to use TinyHooks after destroying it (or outside of an `it`)?

Reply:
Hehe, I was actually planning to ask that at standup, as I was a little undecided.

Reply:
Decided to make the change anyways. Let me know what you think...